src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -24,12 +24,31 @@
 #include "distributed/deparser.h"
 #include "distributed/log_utils.h"
 #include "parser/parse_type.h"
-
+#include "distributed/listutils.h"
 
 static void AppendAlterDatabaseOwnerStmt(StringInfo buf, AlterOwnerStmt *stmt);
 static void AppendAlterDatabaseStmt(StringInfo buf, AlterDatabaseStmt *stmt);
 static void AppendDefElemConnLimit(StringInfo buf, DefElem *def);
 
+const struct option_format create_database_option_formats[] = {
+	{ "template", " TEMPLATE %s", "string" },
+	{ "owner", " OWNER %s", "string" },
+	{ "tablespace", " TABLESPACE %s", "string" },
+	{ "connection_limit", " CONNECTION LIMIT %d", "integer" },
+	{ "encoding", " ENCODING %s", "literal_cstr" },
+	{ "locale", " LOCALE %s", "literal_cstr" },
+	{ "lc_collate", " LC_COLLATE %s", "literal_cstr" },
+	{ "lc_ctype", " LC_CTYPE %s", "literal_cstr" },
+	{ "icu_locale", " ICU_LOCALE %s", "literal_cstr" },
+	{ "icu_rules", " ICU_RULES %s", "literal_cstr" },
+	{ "locale_provider", " LOCALE_PROVIDER %s", "literal_cstr" },
+	{ "is_template", " IS_TEMPLATE %s", "boolean" },
+	{ "allow_connections", " ALLOW_CONNECTIONS %s", "boolean" },
+	{ "collation_version", " COLLATION_VERSION %s", "literal_cstr" },
+	{ "strategy", " STRATEGY %s", "literal_cstr" },
+	{ "oid", " OID %d", "object_id" },
```

Comment:
Can you order these alphabetically, or in the Postgres documentation order, so it is easier to check, add or remove things?

Reply:
I'd prefer postgres documentation order. That makes it easiest to see if anything is missing.